editor/html/reaction.html

```diff
@@ -284,12 +284,14 @@
                     <span class="collapse"></span>
                     Amount
                   </legend>
-                  <input type="radio" class="crude_amount_mass" value="mass" checked> mass
-                  <input type="radio" class="crude_amount_volume" value="volume"> volume
-                  <span style="padding-left:25px">value</span> <div class="crude_amount_value edittext shorttext floattext"></div>
-                  +/- <div class="crude_amount_precision edittext shorttext floattext"></div>
-                  <div class="crude_amount_units_mass selector" data-proto="Mass_MassUnit"></div>
-                  <div class="crude_amount_units_volume selector" data-proto="Volume_VolumeUnit" style="display: none;"></div>
+                  <div>
```

Comment:
Note: wrapping the contents of a collapsible fieldset is necessary to prevent the contents from getting weird and out of order when you collapse/uncollapse. This maybe should have been a separate PR bugfix

Reply:
SGTM